applications/nrf_desktop/doc/user_guide.rst

```diff
@@ -0,0 +1,119 @@
+.. _user_guide:
+
+Application user guide
+######################
+
+The nRF52 Desktop devices can be used to provide the user input to the host using USB or BLE in the same way as other mice and keyboards.
+The devices may also provide the input through the nRF52 Desktop dongle.
+The dongle forwards the data from the Bluetooth connected Peripherals to the host through the USB.
+The Peripherals may be connected using standard BLE or LLPM connection parameters.
+The LLPM enables sending data with high report rate (up to 1000 reports per second), which is not supported by the BLE standard.
+The LLPM is proprietary Bluetooth extension from Nordic Semiconductor and may be used only if it is supported by both connected devices (desktop mice do not support it).
+
+USB connection
+==============
+
+The devices use USB HID class - no additional software or drivers are required.
+The devices connected wirelessly and through the USB at the same time will provide the input only through the USB.
+After USB disconnection, the device will automatically switch to sending the data wirelessly (using Bluetooth).
+Battery powered devices are charged via the USB.
+More detailed information is available in the :ref:`usb_state` module documentation.
+
+Bluetooth connection
+====================
+
+When turned on, the nRF52 Desktop Peripherals are advertising until they go to suspended state or connect via the Bluetooth.
+The Peripheral supports one wireless connection at a time, but it may be bonded with multiple peers.
+
+The nRF52 Desktop Central scans for all the bonded Peripherals that are not connected.
+The scanning is interrupted if any dongle connected device is in use - it would negatively affect user experience (causes report rate drop).
+The scanning for new peers will not be started automatically if all the bonded peers are connected.
+
+More detailed information about the Bluetooth in the nRF52 Desktop is available in the :ref:`bluetooth_guide`.
+
+BLE Peer control
+================
+
+The connected BLE peer can be controlled using predefined button or button combination:
+
+* gaming mouse - "aim button" located on the left side of the mouse.
+* desktop mice - press scroll.
```

Comment:
scroll button (active only for 5 seconds after startup)

Reply:
done